guides/common/modules/proc_installing-openscap-plugin.adoc

```diff
@@ -1,32 +1,31 @@
 [id="Installing_the_OpenSCAP_Plugin_{context}"]
 = Installing the OpenSCAP Plug-in
 
-You can install and enable the OpenSCAP plug-in to generate OpenSCAP compliance reports.
-The OpenSCAP plug-in consists of the main OpenSCAP plug-in itself, the OpenSCAP smart proxy plug-in, and the OpenSCAP Hammer CLI plug-in.
+You can install and enable the OpenSCAP plug-in to generate SCAP compliance reports.
+The OpenSCAP plug-in consists of the OpenSCAP {Project} plug-in, the OpenSCAP Hammer CLI plug-in, the OpenSCAP {SmartProxy} plug-in and modules for policy deployment.
 
 .Procedure
 . Install the OpenSCAP plug-in on your {ProjectServer}:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --enable-foreman-plugin-openscap --enable-foreman-proxy-plugin-openscap --foreman-proxy-plugin-openscap-puppet-module true
+# {foreman-installer} --enable-foreman-plugin-openscap \
```

Comment:
```suggestion
# {foreman-installer} \
--enable-foreman-plugin-openscap \
```

Reply:
@maximiliankolb Perhaps we should move this module to the InstallServer guide. What do you think?

Reply:
IMHO it should be part of the OpenSCAP guide; so I'd say keep it here.